src/analyses/wrapperFunctionAnalysis.ml

```diff
@@ -131,8 +131,19 @@ module MallocWrapper : MCPSpec = struct
       CilType.Location.show loc
 
     let name_varinfo (t, node, c) =
-      Format.asprintf "(alloc@sid:%s@tid:%s(#%s))" (Node.show_id node) (ThreadLifted.show t) (UniqueCount.show c)
-
+        let uniq_count =
+          if UniqueCount.is_top c then
+            Format.dprintf ""
+          else
+            Format.dprintf "(#%s)" (UniqueCount.show c)
+        in
+        let tid =
+          if ThreadLifted.is_top t then
```

Comment:
I'd argue the same here: If we are attempting to compute useful thread ids, I'd still make the top explicit.